frontend/src/components/link/index.tsx

```diff
@@ -81,12 +82,13 @@ export const Link: React.FC<LinkProps> = ({
   url,
   external = false,
   params = null,
+  locale,
   ...additionalProps
 }) => {
   const language = useCurrentLanguage();
 
   if (!url) {
-    url = path.replace("[lang]", language);
+    url = path;
```

Comment:
I am writing this down too. I think the link component is a bit messy right now with lots of href/target/variant/path that are not documented and hard to understand